sqlserver/tests/test_statements.py

```diff
@@ -643,11 +643,13 @@ def test_statement_reported_hostname(
 @pytest.mark.integration
 @pytest.mark.usefixtures('dd_environment')
 def test_statement_basic_metrics_query(datadog_conn_docker, dbm_instance):
-    with datadog_conn_docker.cursor() as cursor:
-        cursor.execute('DBCC FREEPROCCACHE')
+    # with datadog_conn_docker.cursor() as cursor:
+    #     cursor.execute('DBCC FREEPROCCACHE')
```

Comment:
Can we remove this or should it be uncommented?

Reply:
I removed it now.